docs/reference/index-modules.asciidoc

```diff
@@ -76,14 +76,16 @@ breaking change].
 
     The +default+ value compresses stored data with LZ4
     compression, but this can be set to +best_compression+
-    which uses {wikipedia}/DEFLATE[DEFLATE] for a higher
-    compression ratio, at the expense of slower stored fields performance.
+    which uses {wikipedia}/Zstd[ZSTD] for a higher
+    compression ratio, at the expense of slower stored fields read performance.
     If you are updating the compression type, the new one will be applied
     after segments are merged. Segment merging can be forced using
     <<indices-forcemerge,force merge>>. Experiments with indexing log datasets
-    have shown that `best_compression` gives up to ~18% lower storage usage in
-    the most ideal scenario compared to `default` while only minimally affecting
-    indexing throughput (~2%).
+    have shown that `best_compression` gives up to ~28% lower storage usage and
+    better indexing throughput (up to ~10%) in the most ideal scenario compared
+    to `default` while affecting get by id latencies up to 10%. The higher get
```

Comment:
only 10% sounds surprising?

Reply:
Right, maybe too optimistic. Based on [tsdb get by id visualizations](https://elasticsearch-benchmarks.elastic.co/#tracks/tsdb/nightly/default/30d), this is a higher (at least 12%). Let me update this as well to be more realistic.